### PR TITLE
[WIP] This change introduces new features in a generic jobs (ports, exec probes, metadata labels).

### PR DESCRIPTION
**Pull request by ss7pro**
* container ports
* readiness probe exec command
* custom metadata labels

### CODE REVIEW

File: templates/generic/job.yml.j2 [hunk 1]
```diff
@@ -45,6 +48,25 @@ spec:
...
+          ports:
+{%- if ports != [] %}
+    {%- for port in ports %}
+          - containerPort: {{ port['target_port'] }}
+            name: {{ service_name.split('-')[0] }}{{ port['target_port'] }}
+            protocol: TCP
+    {%- endfor %}
+{%- endif %}
+          readinessProbe:
+    {%- if readiness_exec %}
```
Comment (DTadrzak) [inline templates/generic/job.yml.j2]: Move this jinja if to the beginning of a line.

File: templates/generic/job.yml.j2 [hunk 2]
```diff
@@ -45,6 +48,25 @@ spec:
...
+            exec:
+              command:
+              {%- for exec_line in readiness_exec %}
+              - {{ exec_line }}
+              {%- endfor %}
+    {%- elif ports != [] %}
```
Comment (DTadrzak) [inline templates/generic/job.yml.j2]: Move this jinja elif to the beginning of a line.

File: templates/generic/job.yml.j2 [hunk 3]
```diff
@@ -45,6 +48,25 @@ spec:
...
+            tcpSocket:
+              port: {{ ports[0]['target_port'] }}
+    {%- endif %}
```
Comment (DTadrzak) [inline templates/generic/job.yml.j2]: Move this jinja endif to the beginning of a line.

**Comment by PiotrProkop** [comment]
@ss7pro outdated  ? if so please close

**Pull request closed without merging**